### DISCUSSION TITLE
Nothing but "The language model did not provide any assistant messages" today

### DISCUSSION THREAD

**Discussion by themightychris** [cline/cline#864] [Feature Requests]
I'm getting almost nothing but this today with OpenRouter+Sonnet 3.5, am I the only one?

```
Unexpected API Response: The language model did not provide any assistant messages. This may indicate an issue with the API or the model's output.
```

<img width="390" alt="Screenshot 2024-12-06 at 11 04 55 AM" src="https://github.com/user-attachments/assets/25f0ee4d-b3b5-4be4-a72e-0287b08931bf">

**Comment by byjlw** [comment]
Started getting this today. Everything was working fine yesterday.

**Reply by cstacks** [reply]
same

**Comment by elemental-mind** [comment]
As this is not a feature request, can this be moved to an issue or closed?

**Comment by Kaddtechnologies** [comment]
Getting this today.
